Client/WebFilters/ContentBlocker/ContentBlockerRegion.swift

```diff
@@ -30,6 +30,9 @@ extension ContentBlockerRegion {
             log.error("No locale string was provided")
             return nil
         }
+
+        let resourceForLocaleExists = ResourceLocale(rawValue: code) != nil
+        if !resourceForLocaleExists { return nil }
```

Comment:
Why not just `if ResourceLocale(rawValue: code) == nil { return nil }` or `guard ResourceLocale(rawValue: code) else { return nil }`

I'm not sure splitting it up helps readability much :) Maybe better to just add a comment above single line check?

Reply:
Yeah I debated adding a comment vs a readable constant. I'll change it